doc/guide/Guide_Configuration.ipynb

```diff
@@ -30,7 +30,8 @@
     "tags": []
    },
    "source": [
-    "## 1. <a id=\"Const\">Constants</a>\n",
+    "\n",
+    "## 1. <a id=\"Const\"> Constants </a>\n",
```

Comment:
For the notebook itself it doesn't matter much, but sphinx (which is part of the documentation build) doesn't like too many spaces in a title.
```suggestion
    "## 1. <a id=\"Const\">Constants</a>\n",
```

Reply:
I read in the *[Markdown guide](https://www.markdownguide.org/basic-syntax/#code)* that it was best to put a blank line before and after the heading `#` and I thought it might be the reason of the wrong display in the *[website](https://climada-python.readthedocs.io/en/latest/guide/Guide_Euler.html#clone)* (Which was not...) Should I remove those blank lines for each `#heading` ? 

Also, in the last commit I switched to only markdown like this:`[Access to Euler](#access-to-euler)` and `##Access to Euler` for the table of content and cell respectively, since @peanutfun suggested that the mix of markdown and html might produce the wrong outline in the website. I just checked and it doesn't seem to change.

The main issue is that  the headings with two `##` are not displayed, only headings with three `###`. 
I will try few exploratory commits just to see how it changes when I mess with the table of contents.

Do you have any ideas of a possible reason @emanuel-schmid @peanutfun ? 
Thank you!

Reply:
@emanuel-schmid I changed the headers to this style: `# 1. My header`, so there should not be the problem of spaces now

Reply:
Sorry but I don't quite understand... 😕

Reply:
At first the structure of the headers were like this:

`"## 1. <a id=\"link_to_my_header\"> My header name </a>`

where the link_to_my_header is written in the content section like this: 

`[My header name](#link_to_my_header)`

Since with Lukas we thought that it might have caused the problem on the website I switched to only markdown like this:

`"## 1. My header name `

and in the content:

`[My header name](#my-header-name)`

Therefore, the problem you mention of having spaces in the header like this: `## 1. <a id=\"Const\"> Constants </a>\n` is indirectly also resolved.

Reply:
Sorry I missed what the "problem on the website" is. Is it that the links are not working?
Are they working after the link notation is removed - or are they just not displayed as links anymore?
If they are just not displayed as links on the website anymore - it's a pity somehow since they used to be working when you opened the notebook in jupyter.

Reply:
Sorry I wasn't clear with what the problem was, the links worked before and works still now. The problem is (was) that in the website the contents were rendered wrong as you can see *[here](https://climada-python.readthedocs.io/en/latest/guide/Guide_Euler.html )* on the top right corner under `Content`. After the changes it looks right: *[website after changes](https://climada-python--622.org.readthedocs.build/en/622/guide/Guide_Euler.html )*.